analytics/build.gradle

```diff
@@ -3,6 +3,6 @@ apply from: "../config/quality/gitleaks/build.gradle"
 apply plugin: 'kotlin-android'
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
are these still necessary? I mean, does the SDK still need to manually set the compatibility even if the `config` already set it to Java 11?

Reply:
You are right, only Kotlin JvmTarget needs the manually set. Updated.

Reply:
The app can still run and receive the in-app messages when compiling/targeting java 8. I removed the Breaking change note.